Obstacle_Voidance/include/UltrasonicSensors_1.h

```diff
@@ -0,0 +1,21 @@
+// UltrasonicSensors_1.h
+
+//For detecting any objects. blind zone distance of 3cm, super low, it has a detection range of 3cm-4.5m
+//https://www.amazon.com/JSN-SR04T-Integrated-Transducer-Ultrasonic-Waterproof/dp/B0C73XZRFQ/ref=sr_1_5?crid=L86R36BB1MQ1&keywords=waterproof+ultrasonic+sensor&qid=1700536333&sprefix=waterproof+ultras%2Caps%2C166&sr=8-5
+
+
+#ifndef ULTRASONICSENSORS_1_H
+#define ULTRASONICSENSORS_1_H
+
+class UltrasonicSensor {
```

Comment:
Perhaps making these class names more distinctive, such as US_JSN and US_Taidacent rather than ultrasonic1 and 2?